lib/broccoli/ember-app.js

```diff
@@ -291,7 +294,11 @@ EmberApp.prototype._initVendorFiles = function() {
     jquery           = ember.paths.jquery;
   } else {
     jquery = this.bowerDirectory + '/jquery/dist/jquery.js';
-    emberShims = this.bowerDirectory + '/ember-cli-shims/app-shims.js';
+
+    if (emberCliShimsFromBower) {
+      emberShims = this.bowerDirectory + '/ember-cli-shims/app-shims.js';
```

Comment:
No `path.join` would use system separator, so on windows it would be `\` and on posix it would be `/`.  We want it to always be `/`.